ts/src/helpers.cpp

```diff
@@ -279,12 +279,17 @@ std::map<std::string, FREQ_SETTINGS> helpers::parseFrequencies(const std::string
     std::vector<std::string> v = split(sub, ',');
     for (const std::string& xs : v) {
         std::vector<std::string> parts = split(xs.substr(1, xs.length() - 2), '|');
-        if (parts.size() == 3 || parts.size() == 4) {
+        if (parts.size() == 3 || parts.size() == 4 || parts.size() == 5) {
```

Comment:
If that's the case, couldn't we drop `parts.size() == 3` as well? Or is there anything else beside `TFAR_fnc_sendFrequencyInfo` triggering the frequency parsing? A quick search did not releave any special results, but I might have overlooked something.